plugins/script-node-step-plugin/build.gradle

```diff
@@ -0,0 +1,54 @@
+plugins {
+    id 'groovy'
+}
+
+apply from: "../../gradle/java-version.gradle"
+
+ext.pluginClassNames='org.rundeck.plugin.scriptnodestep.ScriptFileNodeStepPlugin,org.rundeck.plugin.scriptnodestep.ScriptNodeStepPlugin,org.rundeck.plugin.scriptnodestep.CommandNodeStepPlugin'
+ext.pluginName = 'Script Node Step Plugin'
+ext.pluginDescription = 'Runs a script file from path or remote URL on a node'
+
+configurations{
+    pluginLibs
+
+    //declare compile to extend from pluginLibs so it inherits the dependencies
+    implementation {
+        extendsFrom pluginLibs
+    }
+}
+
+dependencies {
+    // Use the latest Groovy version for building this library
+    implementation "org.codehaus.groovy:groovy:${groovyVersion}"
+//    implementation "org.rundeck:rundeck-core:${project.rundeckVersion}"
+    pluginLibs group: 'com.esotericsoftware.yamlbeans', name: 'yamlbeans', version:'1.13'
```

Comment:
question: is this used anywhere? should avoid adding new dependencies. if yaml is needed, we normally use Snakeyaml